lib/shared_print/update_record.rb

```diff
@@ -54,7 +56,13 @@ def cast(key, val)
       when :bool
         {"true" => true, "false" => false}[val]
       when :array
-        val.split(",").map(&:strip)
+        split_vals = val.split(",").map(&:strip)
+        if key == :policies
+          if Clusterable::Commitment.incompatible_policies?(split_vals)
```

Comment:
There might be something to extract here about commonalities between update & load validation, but maybe that's only incidental? Like, is this something that should *always* be true about *any* commitment, or is it something particular to phase 3? If the former, maybe it makes sense to put this on `Commitment` itself rather than here.. ? In any case, if in the future we find ourselves making parallel updates here and in `phase_3_validator` we can rethink.